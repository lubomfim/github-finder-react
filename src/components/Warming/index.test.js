import React from 'react';
import { render } from '@testing-library/react';

import { Warming } from './index';

const WarmingComponent = () => {
  return <Warming type="test" />;
};

it('should render correctly', () => {
  const { asFragment } = render(<WarmingComponent />);

  expect(asFragment(<WarmingComponent />)).toMatchSnapshot();
});
