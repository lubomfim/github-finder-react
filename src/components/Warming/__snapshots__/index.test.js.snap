// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  .c0 {
  text-align: center;
}

.c1 {
  font-size: 1.6rem;
  font-weight: 400;
}

<div
    class="c0"
  >
    <p
      class="c1"
    >
      Server offline. Try again later, please.
    </p>
  </div>
</DocumentFragment>
`;
